src/components/HomeMasthead/ImageMasthead.tsx

```diff
@@ -0,0 +1,53 @@
+import Image from 'next/image';
+import blogPlaceholder from '/public/images/blog-placeholder.jpg';
+import LogoImage from '/public/images/fresh-beginnings-logo.png';
+import michelleHome from '/public/images/michelle-home-min.jpg';
+import booksPlaceholderLg from '/public/images/books-placeholder-lg.jpg';
```

Comment:
Some nitpicky things here.
- I like separating my external imports from my internal imports, with external at the top. 
- Lets be consistent with using camel case for our image imports. 
```suggestion
import Image from 'next/image';

import blogPlaceholder from '/public/images/blog-placeholder.jpg';
import logoImage from '/public/images/fresh-beginnings-logo.png';
import michelleHome from '/public/images/michelle-home-min.jpg';
import booksPlaceholderLg from '/public/images/books-placeholder-lg.jpg';
```